tests/cli/ursula/test_run_ursula.py

```diff
@@ -172,14 +172,15 @@ def test_persistent_node_storage_integration(click_runner,
     assert result.exit_code == 0
 
 
+# TODO: test can probably be removed (--dev is not a valid option for `init`)
```

Comment:
I agree, I think it can be safely removed even in this PR

Reply:
✔️